packages/main/src/Badge.ts

```diff
@@ -63,6 +87,65 @@ class Badge extends UI5Element {
 	@property({ defaultValue: "1" })
 	colorScheme!: string;
 
+	/**
+	 * Defines the value state of the component.
+	 *
+	 * @type {sap.ui.webc.base.types.ValueState}
+	 * @defaultvalue "None"
+	 * @name sap.ui.webc.main.Badge.prototype.valueState
+	 * @public
+	 * @since 1.19
+	 */
+	@property({ type: ValueState, defaultValue: ValueState.None })
+	valueState!: `${ValueState}`;
+
+	/**
```

Comment:
can we combine this with colorScheme property. Both actually have colors from 1 to 10 and we can reuse the same enumeration?